moveit_planners/ompl/ompl_interface/src/model_based_planning_context.cpp

```diff
@@ -551,7 +568,7 @@ bool ompl_interface::ModelBasedPlanningContext::solve(planning_interface::Motion
     getSolutionPath(*res.trajectory_.back());
 
     // simplify solution if time remains
-    if (simplify_solutions_ && ptime < request_.allowed_planning_time)
+    if (simplify_solutions_)
```

Comment:
According to @zkingston's above remark, this additional call to `simplifySolution` can be removed, can't it?

Reply:
Not sure if I follow: the call here and where @zkingston commented are a part of different functions, so a single call to `solve` will only end up calling `simplifySolution` once. 

zkingston's comment is clarifying that OMPL now will always attempt to simplify the solution at least once, and that this deleted `ptime < request_.allowed_planning_time` is redundant to what OMPL is now doing internally.

Reply:
Sorry, you are right. I missed the fact that two different `solve` functions are involved - wasn't obvious from github's diff.
I suggest to change the comment in line 570 accordingly.